### ISSUE TITLE
gfortran 4.9.2 compiler error

### ISSUE THREAD

**Issue by migueldvb** [piernik-dev/piernik#110]
Using gfortran 4.9.2 and the compiler flags in gnu47.in I get this error message for the file grid_container.F90:

```
mpif90  -fdefault-real-8 -ffree-form -std=gnu -fimplicit-none -ffree-line-length-none -g -O2 -fstack-arrays -c grid_container.F90
grid_container.F90: In function ‘__final_grid_cont_Grid_container’:
grid_container.F90:1280:0: internal compiler error: in gfc_conv_descriptor_data_get, at fortran/trans-array.c:145
 end module grid_cont
```

**Comment by Xarthisius** [comment]
Yeah, it's [PR59765](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=59765). Should be fixed in gcc-4.9.3, we've been forced to use 4.8.x for that reason

**Comment by migueldvb** [comment]
This error is fixed in gfortran 4.9.2-4 that is available in archlinux:

https://projects.archlinux.org/svntogit/packages.git/tree/trunk?h=packages/gcc
